src/components/Map/VectorTileLayer.tsx

```diff
@@ -27,12 +27,17 @@ export default function VectorTileLayer({ countries, disputedAreas, ipcData, nut
       { countries, disputedAreas, ipcData, selectedMapType, nutritionData },
       mapContainer
     );
-    const popover = new mapboxgl.Popup({ closeButton: false, closeOnClick: false });
+    const popover = new mapboxgl.Popup({
```

Comment:
can you also make this popover sticky like in other map types?

Reply:
@bohdangarchu should be fixed - please test it again on your setup